tests/common.go

```diff
@@ -357,19 +357,40 @@ func ValidateRestoredApplications(contexts []*scheduler.Context, volumeParameter
 func TearDownContext(ctx *scheduler.Context, opts map[string]bool) {
 	context("For tearing down of an app context", func() {
 		var err error
+		var originalSkipClusterScopedObjects bool
 
+		if opts != nil {
+			// Save original value of SkipClusterScopedObjects, if it exists
+			if val, ok := opts[SkipClusterScopedObjects]; ok {
+				originalSkipClusterScopedObjects = val
+			}
```

Comment:
this can be made more succinct. Don't need to the `ok` check. If absent, originalSkipClusterScopedObjects will stay false.

`originalSkipClusterScopedObjects = opts[SkipClusterScopedObjects]`